src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .filter(age -> age >= 30 && age <= 45)
```

Comment:
```suggestion
               .filter(User::허용검증)
```
로직의 의미를 나타내는 메소드를 분리후 해보는 방법도 있을것 같네요!